salt/_roster/kubernetes_nodes.py

```diff
@@ -0,0 +1,55 @@
+import fnmatch
+import logging
+log = logging.getLogger(__name__)
+
+try:
+    import kubernetes.client
+    import kubernetes.config
+    HAS_DEPS = True
+except ImportError:
+    HAS_DEPS = False
+
+__virtualname__ = 'kubernetes'
+
+def __virtual__():
+    if HAS_DEPS:
+        return HAS_DEPS
+    else:
+        return False, 'Unable to load the kubernetes nodes roster: dependencies unavailable.'
```

Comment:
minor: could add the names in the error message for an easier troubleshooting.

Reply:
Done.